examples/main/main.cpp

```diff
@@ -907,7 +907,7 @@ int main(int argc, char ** argv) {
 
         // end of generation
         if (!embd.empty() && llama_token_is_eog(model, embd.back()) && !(params.interactive)) {
-            LOG(" [end of text]\n");
+            LOG_INF(" [end of text]\n");
```

Comment:
This should remain `LOG`, otherwise it will print the log prefix (e.g. timestamp and/or log type) on the same line as the previous tokens